2024-01-17 05:57:21,079:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-01-17 05:57:21,079:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-01-17 05:57:21,079:DEBUG:certbot._internal.main:Arguments: ['--webroot', '-w', '/var/www/certbot', '--email', 'tjfgkssk9472@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'tairot.online']
2024-01-17 05:57:21,079:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-01-17 05:57:21,435:DEBUG:certbot._internal.log:Root logging level set at 30
2024-01-17 05:57:21,444:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-01-17 05:57:21,446:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f3ff3847730>
Prep: True
2024-01-17 05:57:21,447:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f3ff3847730> and installer None
2024-01-17 05:57:21,448:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-01-17 05:57:21,524:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1519022676', new_authzr_uri=None, terms_of_service=None), 053f50e92556dc2ae6fb1958ad04b4cb, Meta(creation_dt=datetime.datetime(2024, 1, 16, 8, 39, 49, tzinfo=<UTC>), creation_host='1b0b13fa81e7', register_to_eff=None))>
2024-01-17 05:57:21,526:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-01-17 05:57:21,530:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-01-17 05:57:22,018:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-01-17 05:57:22,020:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 05:57:21 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "e348b8TFhNM": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-01-17 05:57:22,024:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for tairot.online
2024-01-17 05:57:22,031:DEBUG:acme.client:Requesting fresh nonce
2024-01-17 05:57:22,032:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-01-17 05:57:22,188:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-01-17 05:57:22,189:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 05:57:21 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFKy38ROa1CWYmGJbm5dLlRPt99WA8lqfhCD4SFW8eE5I
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-01-17 05:57:22,190:DEBUG:acme.client:Storing nonce: 70SDU6HFKy38ROa1CWYmGJbm5dLlRPt99WA8lqfhCD4SFW8eE5I
2024-01-17 05:57:22,191:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "tairot.online"\n    }\n  ]\n}'
2024-01-17 05:57:22,193:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRkt5MzhST2ExQ1dZbUdKYm01ZExsUlB0OTlXQThscWZoQ0Q0U0ZXOGVFNUkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "euKq7SpCAKEQDMXLODNBq2fUVTHsBrkPVuwopEGmeJAtWNCFn5xQnpn9oe259AeBGm-xN6aQ-0_JUcJ1mcuHzNr52QeZ1kkQAbNv5AwZPkGiV3fsFlEzI5PtwoOahsKIYaWoFX0T1xbva1_PEbWv8TqTcCaD3alwnzBJ2q_F6UnZz4aQB6p0LQ_LChqStnILBGVQi3aA4oERtt1sTu4cnLL4zcFimEWrrAuiqbD13g6lIhXfAUlliDw3A9Ov2z1tFWff4cEmkjKdv4E4BwQezWmDhtSOzq1HClCGuXSU3on_BE_lfJrbPo0j7cGmiu9PvPkOblbIOfo7cbfADQIsyg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInRhaXJvdC5vbmxpbmUiCiAgICB9CiAgXQp9"
}
2024-01-17 05:57:22,384:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 339
2024-01-17 05:57:22,386:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Wed, 17 Jan 2024 05:57:21 GMT
Content-Type: application/json
Content-Length: 339
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1519022676/237287865426
Replay-Nonce: 70SDU6HF1RLiGoQBVnqNTTDqD65qMwKCBMqrhc6Qt0wM2nKQkyU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-01-24T05:57:21Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "tairot.online"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/305049552316"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1519022676/237287865426"
}
2024-01-17 05:57:22,386:DEBUG:acme.client:Storing nonce: 70SDU6HF1RLiGoQBVnqNTTDqD65qMwKCBMqrhc6Qt0wM2nKQkyU
2024-01-17 05:57:22,387:DEBUG:acme.client:JWS payload:
b''
2024-01-17 05:57:22,389:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305049552316:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRjFSTGlHb1FCVm5xTlRURHFENjVxTXdLQ0JNcXJoYzZRdDB3TTJuS1FreVUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTA0OTU1MjMxNiJ9",
  "signature": "JourGfe8n5k-6K8rBohxKA3yuy4v_K_kRJTTxM0HT1aqgfEzl5NiIcWk1aLYflCxnui9mzrPT0WKouwDoh-F52n1OlZWqN9tZW_CO-bc6pZc7qFR9Z9F7r11ThD7g-PUwIO2uSgFTqUbbUQgymb46Xb9TrgKQU6o2QhHXGqP6DkVjpOC_54_DN25hyX_VtN446LqKbvun7uDhXzDl_27Cy0l7eqIee-s01Yar9PfA3mlbuUEjCgySfaPwIdEa58DjL0Jz7XVwU6kTZRkMiPdx3i6egbxwHnNzzgCKhstN5fbMj5JCc3E0l-Y28-jtT2Vd9zBR3Hs34dDfHKITyoCWg",
  "payload": ""
}
2024-01-17 05:57:22,574:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305049552316 HTTP/1.1" 200 797
2024-01-17 05:57:22,576:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 05:57:21 GMT
Content-Type: application/json
Content-Length: 797
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFtnR_SmjI5ILek6WMKugC_v-ovHRtjDSvvgMP7yncVWw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "pending",
  "expires": "2024-01-24T05:57:21Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/iof1yg",
      "token": "CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/xewkNg",
      "token": "CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/_L908Q",
      "token": "CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU"
    }
  ]
}
2024-01-17 05:57:22,576:DEBUG:acme.client:Storing nonce: 70SDU6HFtnR_SmjI5ILek6WMKugC_v-ovHRtjDSvvgMP7yncVWw
2024-01-17 05:57:22,578:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-01-17 05:57:22,578:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 05:57:22,579:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2024-01-17 05:57:22,580:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2024-01-17 05:57:22,590:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU
2024-01-17 05:57:22,594:DEBUG:acme.client:JWS payload:
b'{}'
2024-01-17 05:57:22,595:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/iof1yg:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRnRuUl9TbWpJNUlMZWs2V01LdWdDX3Ytb3ZIUnRqRFN2dmdNUDd5bmNWV3ciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMwNTA0OTU1MjMxNi9pb2YxeWcifQ",
  "signature": "j_djgJm9lo5VYaZEEJX4WHMJT_uD5CIFiaiz4fls5_GeWrlPnPYydMWobPOZiPMkCqakYYAbRmvDXZh_WFNqIX_qsOD4uW2FoMb2RQ7EIVYq4nK5FclQCc77rDs0c-klhkJmuvnKxQf7sKUKR0oNqtLXQbBscPsOIa99HUytTrW2ZSsH_ux81Uu_tx-uKRw_eYxr-WHXpdP4PtKjWnSNy3B0wMNqQl-GlP0kEJGcBrV03D7dTSncRFQYvwWW5cNYOuS9QyW2F8YivAi_jxCf0_bkIdu6iLgf-Dh1VnXHLAzYFsfNHTrIikwva_VJ8VWtyvJmC8kpfsZXUWf6PmQuMA",
  "payload": "e30"
}
2024-01-17 05:57:22,754:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/305049552316/iof1yg HTTP/1.1" 200 187
2024-01-17 05:57:22,755:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 05:57:22 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/305049552316>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/iof1yg
Replay-Nonce: 70SDU6HFHim79_gKh2_wBxzPNYKeoX-4a248l0S3IcR1r5NdWF0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/iof1yg",
  "token": "CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU"
}
2024-01-17 05:57:22,756:DEBUG:acme.client:Storing nonce: 70SDU6HFHim79_gKh2_wBxzPNYKeoX-4a248l0S3IcR1r5NdWF0
2024-01-17 05:57:22,757:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-01-17 05:57:23,759:DEBUG:acme.client:JWS payload:
b''
2024-01-17 05:57:23,760:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305049552316:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRkhpbTc5X2dLaDJfd0J4elBOWUtlb1gtNGEyNDhsMFMzSWNSMXI1TmRXRjAiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTA0OTU1MjMxNiJ9",
  "signature": "YIftGcB7fB4Krwcgfo6GbedVNoa0Z-kyUP8yUVBDyJicN6AmcUAGK6oj2bG2FmbgwmGebqSmOlBU2l7-QG_3ISXKDBa085--QTtqYIhjWdfWpmxDgODEPJE4qrVKf48yOx64plI4ziiF2BJkKKGRmseWYi9A3R7va8-lbLbzE0bqnnRN4zDT32TF2-bKw3iqERZjZJyOfIvMjsrywlwDFgGjfgT5Kzkat3SIVUQC8TnKWt4A9ZO5kN8YjNhDNrTW8GMUsvo3zjioAxtLTBm5lG9lSyBkONbkBHl2mlKejPd9Q_gc6FeNoo0clJkdQWoAeUw8EoLzIWAbmz4QuFkcxA",
  "payload": ""
}
2024-01-17 05:57:23,922:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305049552316 HTTP/1.1" 200 1023
2024-01-17 05:57:23,923:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 05:57:23 GMT
Content-Type: application/json
Content-Length: 1023
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HF1LaSfhZrq3T_LRhDs8zU-UxtyRqwkHGXmOzN3XhziMk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "invalid",
  "expires": "2024-01-24T05:57:21Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "43.202.208.226: Fetching http://tairot.online/.well-known/acme-challenge/CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU: Connection refused",
        "status": 400
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305049552316/iof1yg",
      "token": "CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU",
      "validationRecord": [
        {
          "url": "http://tairot.online/.well-known/acme-challenge/CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU",
          "hostname": "tairot.online",
          "port": "80",
          "addressesResolved": [
            "43.202.208.226"
          ],
          "addressUsed": "43.202.208.226"
        }
      ],
      "validated": "2024-01-17T05:57:22Z"
    }
  ]
}
2024-01-17 05:57:23,923:DEBUG:acme.client:Storing nonce: 70SDU6HF1LaSfhZrq3T_LRhDs8zU-UxtyRqwkHGXmOzN3XhziMk
2024-01-17 05:57:23,924:INFO:certbot._internal.auth_handler:Challenge failed for domain tairot.online
2024-01-17 05:57:23,924:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 05:57:23,925:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: tairot.online
  Type:   connection
  Detail: 43.202.208.226: Fetching http://tairot.online/.well-known/acme-challenge/CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-01-17 05:57:23,925:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-01-17 05:57:23,926:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-01-17 05:57:23,926:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-01-17 05:57:23,927:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/CwNgioSTWuBoIQ4-WD3u5MqFST6fN7w1rGMNUPIolYU
2024-01-17 05:57:23,931:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-01-17 05:57:23,932:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-01-17 05:57:23,934:ERROR:certbot._internal.log:Some challenges have failed.
